### PR TITLE
Update reuse_garage_policy.md

### PR DESCRIPTION
**Pull request by ahtsan**
Include `tf.Session()` for TensorFlow example

### CODE REVIEW

**Comment by ahtsan** [comment]
Close #1717

**Comment by ahtsan** [comment]
@Mergifyio rebase

File: docs/user/reuse_garage_policy.md [hunk 1]
```diff
@@ -25,8 +25,11 @@ of code:
 ```python
 # Load the policy
 from garage.experiment import Snapshotter
+import tensorflow as tf // optional, only for TensorFlow as we need a tf.Session
```
Comment (ryanjulian) [inline docs/user/reuse_garage_policy.md:28]: // isn't a valid comment in Python :)

**Comment by ahtsan** [comment]
Something weird happens to the branch, will close this PR and open another one.

**Pull request closed without merging**